### ISSUE TITLE
Deduplication

### ISSUE THREAD

**Issue by Zebbelin** [MicrosoftDocs/SCCMdocs#831]
The Tip for "Remote Contentlib" is missleading. It says "If you move the content library to the same storage volume as your package sources, and this volume supports content de-duplication, this design may save the amount of necessary storage" However dedup is not supported for the package source: https://docs.microsoft.com/en-us/sccm/core/plan-design/configs/support-for-windows-features-and-networks#bkmmk_datadedup

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 92260657-2102-c15f-a844-5fa3aff64aae
* Version Independent ID: a4597d2d-9441-61f2-718d-c0a54a46d3fd
* Content: [The content library - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/plan-design/hierarchy/the-content-library)
* Content Source: [sccm/core/plan-design/hierarchy/the-content-library.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/plan-design/hierarchy/the-content-library.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks @Zebbelin, good catch. Let me check on this, stay tuned.

**Comment by aczechowski** [comment]
I confirmed with engineering that this current tip is incorrect: the package sources can't be on a volume marked for data dedupe. I just submitted a revision with a private pull request. Once merged, this change should be live with the next daily publishing process. Right now that *should* be today (Tuesday, 11 September) around 11:00 Pacific time. I'm closing this issue in anticipation of that process. Thanks again for your feedback!
